src/nestjs-back/src/users/entities/blockedUsers.entity.ts

```diff
@@ -0,0 +1,21 @@
+import {
+    Column,
+    Entity,
+    ManyToOne,
+    PrimaryGeneratedColumn
+  } from "typeorm";
+  import { User } from "src/users/entities/users.entity";
+  
+  @Entity()
+  export class BlockedUsers {
+    @PrimaryGeneratedColumn()
+    id: number;
+  
+    @Column({ length: 50, unique: true })
+    name: string;
```

Comment:
I think this one is useless because you can have the username with the user below...